app/javascript/packs/admin.js

```diff
@@ -185,6 +185,12 @@ ready(() => {
   const registrationMode = document.getElementById('form_admin_settings_registrations_mode');
   if (registrationMode) onChangeRegistrationMode(registrationMode);
 
+  const checkAllElement = document.querySelector('#batch_checkbox_all');
```

Comment:
Hm, what are these changes for?

Reply:
It is because the page may be pre-rendered with a mix of unchecked and checked inputs: everything comes pre-checked except for domains where there are existing follow relationships.